src/event/workqueue.c

```diff
@@ -69,7 +69,7 @@ typedef struct dispatch_workq_monitor_s {
 	int num_registered_tids;
 } dispatch_workq_monitor_s, *dispatch_workq_monitor_t;
 
-static dispatch_workq_monitor_s _dispatch_workq_monitors[WORKQ_NUM_PRIORITIES];
+static dispatch_workq_monitor_s _dispatch_workq_monitors[DISPATCH_QOS_MAX+1];
```

Comment:
just to avoid index conversion, but I will change to match convention used elsewhere.